dtschema/schemas/options/u-boot.yaml

```diff
@@ -71,6 +71,21 @@ properties:
       2: use simplified command line (e.g. avoid hush)
       3... reserved
 
+  bootscr-address:
+    $ref: /schemas/types.yaml#/definitions/uint64
+    default: 0x0
+    description:
+      This property holds the full address of the boot script file. This helps in
```

Comment:
nit: s/This property holds/Holds/

Does this get put into the scriptaddr variable? If not, what is the relationship between these two? I think this needs more detail about how to use it. You have written it in the comments below, so please add it here